src/tags/control/DateTime.js

```diff
@@ -0,0 +1,338 @@
+import * as d3 from "d3";
+import React from "react";
+import { observer, inject } from "mobx-react";
+import { types } from "mobx-state-tree";
+
+import InfoModal from "../../components/Infomodal/Infomodal";
+import { guidGenerator } from "../../core/Helpers";
+import Registry from "../../core/Registry";
+import { AnnotationMixin } from "../../mixins/AnnotationMixin";
+import PerRegionMixin from "../../mixins/PerRegion";
+import RequiredMixin from "../../mixins/Required";
+import { isDefined } from "../../utils/utilities";
+import ControlBase from "./Base";
+
+const FORMAT_FULL = "%Y-%m-%dT%H:%M";
+const FORMAT_DATE = "%Y-%m-%d";
+const FORMAT_TIME = "%H:%M";
+
+const ISO_DATE_SEPARATOR = "T";
+
+const zero = n => (n < 10 ? "0" : "") + n;
+
+/**
+ * The DateTime tag adds date and time selection to the labeling interface. Use this tag to add a date, timestamp, month, or year to an annotation.
+ *
+ * Use with the following data types: audio, image, HTML, paragraph, text, time series, video
+ * @example
+ * <View>
+ *   <Text name="txt" value="$text" />
+ *   <DateTime name="datetime" toName="txt" only="date" />
+ * </View>
+ *
+ * @name DateTime
+ * @param {string} name              - Name of the element
+ * @param {string} toName            - Name of the element that you want to label
+ * @param {string} only              - Comma-separated list of parts to display (date, time, month, year)
+ *        date and month/year can't be used together. The date option takes precedence
+ * @param {string} format            - Input/output strftime format for datetime (internally it's always ISO);
```

Comment:
not sure if it matters to point out that "internally it's always ISO" unless that means that ISO is what gets exported. 
Does input/output here mean that it's the formatting for either the input OR the output or the input AND the output? if the latter, plz change `/` to `and`

Reply:
can't add a suggestion but change it to:

strftime-formatted datetime string